ow_plexil/src/plexil-adapter/OwInterface.cpp

```diff
@@ -160,17 +149,20 @@ static map<int, string> GoalStatus {
 static map<string, int> ActionGoalStatusMap {
   // ROS action name -> Action goal status
   // Assigning -1 as the default state
-
-  { "Stow", -1 },
-  { "Unstow", -1 },
-  { "Grind", -1 },
-  { "GuardedMove", -1 },
-  { "ArmMoveJoint", -1 },
-  { "ArmMoveJoints", -1 },
-  { "DigCircular", -1 },
-  { "DigLinear", -1 },
-  { "Deliver", -1 },
-  { "Discard", -1 }
+  { Op_Stow, -1 },
+  { Op_Unstow, -1 },
+  { Op_Grind, -1 },
+  { Op_GuardedMove, -1 },
+  { Op_ArmMoveJoint, -1 },
+  { Op_ArmMoveJoints, -1 },
+  { Op_DigCircular, -1 },
+  { Op_DigLinear, -1 },
+  { Op_Deliver, -1 },
+  { Op_Discard, -1 },
+  { Op_TakePicture, -1 },
+  { Op_PanTiltAntenna, -1 },
+  { Op_IdentifySampleLocation, -1 },
+  { Op_SetLightIntensity, -1 }
 };
```

Comment:
What does the status integer mean? Can it be replaced with an enumerate?

Reply:
Good suggestion, replaced with enum (and also in one more -1 usage).

Furthermore I felt that the one use of the previous map was unnecessary and I removed it, simplifying that function in another way as well.  I retested ReferenceMission1 and the TestAntennaMovement.